frontend/src/components/common/Resource/ScaleButton.tsx

```diff
@@ -163,16 +163,21 @@ function ScaleDialog(props: ScaleDialogProps) {
               <Fab
                 size="small"
                 color="primary"
-                onClick={() => setNumReplicas(numReplicas => numReplicas - 1)}
+                onClick={() => setNumReplicas(numReplicas => Math.max(0, numReplicas - 1))}
                 aria-label={t('translation|Decrement')}
+                disabled={numReplicas <= 0}
               >
                 <Icon icon="mdi:minus" width="22px" />
               </Fab>
               <Input
                 type="number"
                 value={numReplicas}
                 sx={{ marginLeft: '6px', marginRight: '6px' }}
-                onChange={e => setNumReplicas(Number(e.target.value))}
+                onChange={e =>
+                  Number(e.target.value) <= 0
+                    ? setNumReplicas(0)
+                    : setNumReplicas(Number(e.target.value))
```

Comment:
Nit: Could be simpler by doing `setNumReplicas(Math.max(0, Number(e.target.value)))`